<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisualStationClass</class>
 <widget class="QMainWindow" name="VisualStationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1135</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VisualStation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="VisualStation.qrc">
    <normaloff>:/VisualStation/NPU_rLogal.png</normaloff>:/VisualStation/NPU_rLogal.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0" colspan="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>377</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Copyright (C) 2023 西北工业大学-张璞玉. A11 rights reserved.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QStackedWidget" name="stackedWidget_Views">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="stackedWidget_ViewsPage_3D">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="ViewWidget_3D" name="viewWidget_3D" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="stackedWidget_ViewsPage_2D">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="ViewWidget_2D" name="viewWidget_2D" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1135</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
    <addaction name="actionSwitchTo2D"/>
    <addaction name="actionSwitchTo3D"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="actionSwitchTo2D">
   <property name="text">
    <string>二维视图</string>
   </property>
  </action>
  <action name="actionSwitchTo3D">
   <property name="text">
    <string>三维视图</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="actionConnectSocket">
   <property name="text">
    <string>启动Socket</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ViewWidget_3D</class>
   <extends>QWidget</extends>
   <header location="global">viewwidget_3d.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ViewWidget_2D</class>
   <extends>QWidget</extends>
   <header>viewwidget_2d.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="VisualStation.qrc"/>
 </resources>
 <connections/>
</ui>
